tests/helpers/load_state_tests.py

```diff
@@ -378,6 +378,7 @@ def run_blockchain_st_test(test_case: Dict, load: BaseLoad) -> None:
     )
 
     if not test_data["ignore_pow_validation"] or load.fork_module == "paris":
+        # FIXME: Test for a POS hardfork properly, rather than `== "paris"`
```

Comment:
What does the `sealEngine` field represent for post merge fork tests?

Reply:
`sealEngine` is set to `NoProof` everywhere, except in `LegacyTests`. The problem is that our response to seeing `NoProof` is to try and mock a bunch of POW validation functions that don't exist in `paris` and later forks.